docs/resources/rds_instance.md

```diff
@@ -200,6 +200,14 @@ The following arguments are supported:
   -> **NOTE:** async indicates the asynchronous replication mode. semisync indicates the semi-synchronous replication
   mode. sync indicates the synchronous replication mode.
 
+* `lower_case_table_names` - (Optional, String, ForceNew) Specifies the case-sensitive state of the database table name,
```

Comment:
as the parameter can not be imported, please add a notification in Import section.

Reply:
Ok, I have modified it